api-extractor/src/DocItemLoader.ts

```diff
@@ -48,6 +50,8 @@ export default class DocItemLoader {
     }
 
     this._projectFolder = projectFolder;
+    this._rootTypes = ['lib/external-api-json/'];
+    this._rootSuffixes = ['es6-collections', 'es6-promise', 'whatwg-fetch'];
```

Comment:
This is already provided by [ApiExtractorTask.ts](https://github.com/Microsoft/web-build-tools/blob/master/gulp-core-build-typescript/src/ApiExtractorTask.ts)